std_msgs/Header header

sensor_msgs/Image image
sensor_msgs/Image mask
sensor_msgs/Image abs_depth
sensor_msgs/Image rel_depth
float32[2] factor